keras/utils/generic_utils.py

```diff
@@ -208,7 +208,7 @@ class Progbar(object):
     """Displays a progress bar.
 
     # Arguments
-        target: Total number of steps expected.
+        target: Total number of steps expected, -1 if unknown.
```

Comment:
Let's use `None` if unknown, that would be more consistent with the rest of the API.